src/plugins/interactive_setup/server/kibana_config_writer.ts

```diff
@@ -187,7 +187,8 @@ export class KibanaConfigWriter {
    */
   private static getFleetDefaultOutputConfig(caCert: string, host: string): FleetOutputConfig[] {
     const cert = new X509Certificate(caCert);
-    const certFingerprint = cert.fingerprint256;
+    // fingerprint256 is a ":" separated uppercase hexadecimal string
+    const certFingerprint = cert.fingerprint256.split(':').join('').toLowerCase();
```

Comment:
Curious how you tested this to determine we needed to change this. Were you able to test with the Beats changes that were already done?

Reply:
I sent the config to @belimawr and I tested it with the agent standalone seems to work.

Reply:
Got it, thanks. I see this also matches what ES is outputting directly in the logs when first configuring the certificates. For example:

```
The hex-encoded SHA-256 fingerprint of the generated HTTPS CA DER-encoded certificate:
4ec9320d14c967d1e97058d26eb36b058277ef6319d3b0c2833f8580d8f263a2
```